sdk/storage/azure-storage-common/inc/azure/storage/common/shared_key_policy.hpp

```diff
@@ -28,12 +28,12 @@ namespace Azure { namespace Storage { namespace _internal {
 
     std::unique_ptr<Core::Http::RawResponse> Send(
         Core::Http::Request& request,
-        Core::Http::Policies::NextHttpPolicy nextHttpPolicy,
-        Core::Context const& ctx) const override
+        Core::Http::Policies::NextHttpPolicy nextPolicy,
+        Core::Context const& context) const override
```

Comment:
This is great! I've been wanting to uniform the parameter name `ctx` and `context` for a long time, just haven't got time to do it.